### PR TITLE
p31

### PR DESCRIPTION
**Pull request by Sellafield**

### CODE REVIEW

File: src/Perpetuum.Bootstrapper/PerpetuumBootstrapper.cs [hunk 1]
```diff
@@ -272,12 +276,13 @@ public IEnumerable<Command> GetCommands()
 
         public void Init(string gameRoot)
         {
+            Thread.Sleep(10000);
```
Comment (clouths) [inline src/Perpetuum.Bootstrapper/PerpetuumBootstrapper.cs]: Does the server need this sleep when starting to sync with an other service?
Is it required to start the server?

Comment (Sellafield) [reply src/Perpetuum.Bootstrapper/PerpetuumBootstrapper.cs]: Not really, no, just forgotten debug measure. Removed

File: src/Perpetuum/Zones/ZoneConfiguration.cs [hunk 2]
```diff
@@ -104,7 +104,7 @@ public sealed class ZoneConfiguration
         public int RaceId { get; set; }
         public string Note { get; set; }
 
-        public string ListenerAddress { get; set; }
+        public string ListenerAddress { get; set; } = "127.0.0.1";
```
Comment (clouths) [inline src/Perpetuum/Zones/ZoneConfiguration.cs:107]: Does setting a default value here change the behavior of a zone configuration?

**Review by aqpanaciy — APPROVED** [review]

File: src/Perpetuum/Zones/ZoneConfiguration.cs (hunk 2 above)
Comment (Sellafield) [reply src/Perpetuum/Zones/ZoneConfiguration.cs:107]: It doesn't affect anything except some debug cases

**Review by clouths — APPROVED** [review]

**Pull request merged**